### DISCUSSION TITLE
Please, do not drop armhf (armv6) in future releases

### DISCUSSION THREAD

**Discussion by GoodSoulGermany** [userdocs/qbittorrent-nox-static#70] [General]
Hey,

many Rasperry Pie users are depending on your release (offical armhf is extrem outdated). So I beg you to not drop the armhf support. 

Thnx in advance!

**Comment by userdocs** [comment]
It's ok, actually already decided to not drop it.

The script can now build armhf on debian and alpine so i added all the functionally back.

edit:

I updated the readme https://github.com/userdocs/qbittorrent-nox-static#armhf-armv6

So nothing will change for future releases.

**Reply by GoodSoulGermany** [reply]
Thanks, you are really awesome! 🥇

**Reply by Sparklingx** [reply]
awesome
